app/lib/widgets/ToDoListView.dart

```diff
@@ -48,198 +52,373 @@ class _ToDoListViewState extends State<ToDoListView> {
 
   @override
   Widget build(BuildContext context) {
-    return Card(
-      elevation: 0,
-      margin: const EdgeInsets.all(8.0),
-      shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(15.0)),
-      color: ToDoTheme.secondaryColor,
-      child: Obx(
-        () => ExpansionTile(
-          title: Padding(
-            padding: const EdgeInsets.symmetric(vertical: 10, horizontal: 5),
-            child: Text(widget.title, style: ToDoTheme.listTitleTextStyle),
-          ),
-          subtitle: Padding(
-            padding: const EdgeInsets.all(5),
+    return GestureDetector(
+      onTap: () {
+        setState(() {
+          isCardExpanded = !isCardExpanded;
```

Comment:
We already had UI build helper functions in ```todo_controller.dart``` I think.

```
 void toggleExpand() {
    initialExpand.value = !initialExpand.value;
  }

  void toggleExpandBtn() {
    expandBtn.value = !expandBtn.value;
  }
  ```
  Or make one there and call ```GetBuilder()```.